src/util/isEM.ts

```diff
@@ -1,8 +1,11 @@
 import { EM_TOKEN } from '../constants'
-import { Path } from '../types'
+import { Child } from '../types'
 
 /** Returns true if the Path is the EM_TOKEN. */
-export const isEM = (path: Path): boolean =>
-  path.length === 1 &&
-  !!path[0] &&
-  path[0].value === EM_TOKEN
+export const isEM = (thoughts: (string | Child)[]): boolean =>
+  thoughts.length === 1 &&
+  !!thoughts[0] &&
```

Comment:
Since we already know `thought.length === 1`, then we don't need to check `!!thoughts[0]`. An empty thought will be handled by the subsequent case.